packages/app/src/analytic-data-tracker/services/analytic-data-tracker.service.ts

```diff
@@ -37,9 +38,10 @@ export class AnalyticDataTrackerService implements AnalyticDataTrackerClass {
     this.siteId = options.siteId
     this.apiUrl = options.apiUrl
     this.trackingEnabled = options.trackingEnabled ?? true
+    this.externalUsageDevice = false
   }
   public async track(event: TrackEvent): Promise<AxiosResponse | undefined> {
-    if (!this.trackingEnabled) {
+    if (!this.trackingEnabled || !this.externalUsageDevice) {
```

Comment:
🟡 
This check excludes the collection of all events, even those not related to the device. Let's see how to improve this to include compatibility for non-device events (like `CenterVersion`) 😉